package.json

```diff
@@ -33,24 +33,24 @@
     "grunt-contrib-uglify": "~4.0.1",
     "grunt-contrib-watch": "^1.1.0",
     "jit-grunt": "^0.10.0",
-    "sass": "^1.26.3",
+    "sass": "^1.26.5",
     "standard": "^14.3.3"
   },
   "dependencies": {
-    "ajv": "^6.12.0",
+    "ajv": "^6.12.2",
     "animate.css": "^3.7.2",
     "animatedmodal": "^1.0.0",
     "bootstrap": "^4.4.1",
     "clipboard": "^2.0.6",
     "grunt-contrib-uglify-es": "github:gruntjs/grunt-contrib-uglify#harmony",
-    "jquery": "^3.4.1",
+    "jquery": "^3.5.0",
```

Comment:
This will result in failing checks as soon as you have rebased.
https://github.com/Catrobat/Catroweb-Symfony/pull/527